modules/ztp-configuring-the-cluster-for-a-disconnected-environment.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-configuring-the-cluster-for-a-disconnected-environment_{context}"]
+= Configuring a managed cluster for a disconnected environment
+
+After you have completed the preceding procedure, follow these steps to configure the managed cluster for a disconnected environment.
+
+.Prerequisites
+
+* A disconnected installation of Red Hat Advanced Cluster Management (RHACM) 2.3.
```

Comment:
Please use the attributes for RHACM:
{rh-rhacm-first} =  Red Hat Advanced Cluster Management (RHACM)
{rh-rhacm} =  RHACM